src/backend/oneapi/tile.cpp

```diff
@@ -25,8 +25,9 @@ Array<T> tile(const Array<T> &in, const af::dim4 &tileDims) {
 
     Array<T> out = createEmptyArray<T>(oDims);
 
-    ONEAPI_NOT_SUPPORTED("tile Not supported");
-    // kernel::tile<T>(out, in);
+    if constexpr(!(std::is_same_v<T, double> || std::is_same_v<T, cdouble>)) {
```

Comment:
Let's wait for the resolution of #3342 to determine the best approach here

Reply:
umar confirmed this approach. without it, arrayfire outright crashes